src/core/reference/include/openvino/reference/batch_norm.hpp

```diff
@@ -46,5 +45,4 @@ void batch_norm_inference(float eps,
     NGRAPH_SUPPRESS_DEPRECATED_END
```

Comment:
We also can remove legacy NGRAPH macros and headers. It can be done in separate PR.